src/pages/resources/index.ejs

```diff
@@ -14,7 +14,7 @@
     <main class="internal-page__content">
         <header class="hero-background">
             <header class="hero-background">
-                <section class="internal-header mdl-grid mdl-grid--no-spacing">
```

Comment:
let's add `--no-spacing`, `h4` already has `margin: 40px 0;`, otherwise the hero will have additional padding.